samples/tls/monitoring/README.md

```diff
@@ -64,24 +27,49 @@ my_repository_path/monitoring/bin/deploy_monitoring_cluster.sh
 
 ## Notes On Customization Values
 
-Set `MON_TLS_ENABLE=true` in the `$USER_DIR/monitoring/user.env` file. This variable modifies the deployment of Prometheus,
-Grafana, and Alertmanager to be TLS-enabled.
+* Set `TLS_ENABLE=true` in the `$USER_DIR/monitoring/user.env` file to specify that Prometheus, Grafana, and Alertmanager are deployed with TLS enabled for connections to these components. Due to limitations in the underlying Helm charts, some components might not have TLS enabled for in-cluster connections between components. See the **Limitations and Known Issues** section below for details.
 
-Edit `$USER_DIR/monitoring/user-values-prom-operator.yaml` and replace
+* Edit `$USER_DIR/monitoring/user-values-prom-operator.yaml` and replace
 any sample hostnames with hostnames for your deployment. Specifically, you must replace
 `host.cluster.example.com` with the name of the ingress node. Often, the ingress node is the cluster master node, but your environment might be different.
 
-## Limitations and Known Issues
+* Manually populate these TLS secrets for ingress in the `monitoring` namespace (or `MON_NS` value) **before** you deploy cluster monitoring.
```

Comment:
Seems like this item is only relevant if the reader wants to deploy ingress objects, so we probably should call that out.  This sentence and the one that follows don't flow.
Perhaps something like:
_If you plan on using ingress, before you deploy cluster monitoring, you must manually populate specific Kubernetes secrets with TLS certificates.  The following Kubernetes secrets must be populated:
...
You can use the following command to generate the necessary secret for each set of TLS certificates:_

Reply:
Changed as noted